metaci/build/migrations/0015_auto_20180815_2300.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-08-15 23:00
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('release', '0002_auto_20180815_2248'),
+        ('build', '0014_auto_20180601_2003'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='build',
+            name='release',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='release.Release'),
+        ),
+        migrations.AddField(
+            model_name='build',
+            name='release_relationship_type',
+            field=models.CharField(blank=True, choices=[('test', 'Release Test'), ('automation', 'Release Automation')], max_length=50, null=True),
```

Comment:
This is missing the Manual option that's included in models.py

Reply:
Good catch. makemigrations didn't care enough to build a new migration for it, but I'll include it manually.